packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -166,16 +166,20 @@ function ChangePassword({
     if (errorMessage) {
       setErrorMessage(null);
     }
-    try {
-      await changePassword({ user, currentPassword, newPassword });
 
-      onSuccess?.(); // notify success to the parent
-    } catch (e) {
-      const error = e as Error;
-      if (error.message) setErrorMessage(error.message);
+    const promise = changePassword({ user, currentPassword, newPassword });
 
-      onError?.(error); // notify error to the parent
-    }
+    promise
```

Comment:
We can just call `changePassword` directly here instead of assigning it to a variable

Reply:
SGTM